static/templates/bot_avatar_row.handlebars

```diff
@@ -44,6 +44,16 @@
                   <div><label for="edit_bot_name{{id_suffix}}" generated="true" class="text-error"></label></div>
                </div>
               </div>
+              <div class="control-group">
+                <label for="edit_bot_owner{{id_suffix}}" class="control-label">{{t "Owner" }}</label>
+                <div class="controls">
+                  <select name="bot_owner" class="edit_bot_owner" id="edit_bot_owner{{id_suffix}}">
```

Comment:
In general we've been moving away from constructed IDs like this to instead using a `data-user-id` field on the object.  We can probably fix this in a follow-up pass, but I think we should plan to migrate the bots UI to use that.  I opened https://github.com/zulip/zulip/issues/3643 for that issue.